docs/README.md

```diff
@@ -0,0 +1,26 @@
+# HYPR Authenticator
```

Comment:
Have we used sentence case or title caps for headings in the extensions? Hope we are being consistent with that.

When using sentence case, we can use `authenticator` instead of `Authenticator`.
Also, in general running text, we can say `HYPR authenticator` unless `HYPR Authenticator` is  a specific practice.

Reply:
For headings title caps are used.

Reply:
Resolves via [ab5e02d](https://github.com/wso2-extensions/identity-outbound-auth-hypr/pull/2/commits/ab5e02d88c973fd971f23c3253533a26e6bbf16b)